pocs/focuser/focuser.py

```diff
@@ -324,7 +346,10 @@ def _autofocus(self,
                    plots,
                    start_event,
                    finished_event,
-                   smooth=0.4, *args, **kwargs):
+                   smooth,
```

Comment:
The order is is backwards from everywhere else (with smooth first then dilations). I notice also that this is just a positional parameter then the thread passes kwargs. I'm assuming things go in correctly but seems a little odd just since they are not keyword args here. Probably doesn't need to change.

Reply:
Re-ordered for consistency, but yes, it shouldn't matter as I always call it with keyword arguments (for precisely this reason!).